src/rum/instrumentation/DdRumResourceTracking.tsx

```diff
@@ -73,7 +125,8 @@ export class DdRumResourceTracking {
       const traceId = generateTraceId()
       this._datadog_xhr = {
         method,
-        startTime: -1,
+        startTime: MISSING_TIME,
+        loadStartTime: MISSING_TIME,
```

Comment:
can we think to different name here for `startTime` ? I find it hard to understand what is exactly measuring.

Reply:
we could use `requestStartTime`, but then it won't conform the definition in https://developer.mozilla.org/en-US/docs/Web/API/Resource_Timing_API/Using_the_Resource_Timing_API, currently, with such name, it conforms.